src/analytics/mod.rs

```diff
@@ -43,42 +44,38 @@ impl CastAnalytics {
         export_bucket: &str,
         node_ip: IpAddr,
         geoip: GeoIpReader,
-    ) -> Result<Self> {
+    ) -> anyhow::Result<Self> {
         info!(%export_bucket, "initializing analytics with aws export");
 
-        let opts = BatchCollectorOpts::default();
+        let opts = BatchOpts::default();
         let bucket_name: Arc<str> = export_bucket.into();
         let node_ip: Arc<str> = node_ip.to_string().into();
 
         let messages = {
-            let exporter = AwsExporter::new(AwsExporterOpts {
-                export_name: "push_messages",
+            let exporter = AwsExporter::new(AwsOpts {
+                export_prefix: "cast/messages",
+                export_name: "messages",
                 file_extension: "parquet",
-                // Note: Clone these values if we add more exporters
                 bucket_name: bucket_name.clone(),
                 s3_client: s3_client.clone(),
                 node_ip: node_ip.clone(),
             });
 
-            Analytics::new(
-                gorgon::batcher::create_parquet_collector::<MessageInfo, _>(opts.clone(), exporter)
-                    .map_err(|e| Error::BatchCollector(e.to_string()))?,
-            )
+            let collector = ParquetWriter::<MessageInfo>::new(opts.clone(), exporter)?;
+            Analytics::new(collector)
         };
 
         let clients = {
-            let exporter = AwsExporter::new(AwsExporterOpts {
-                export_name: "push_clients",
+            let exporter = AwsExporter::new(AwsOpts {
+                export_prefix: "cast/clients",
```

Comment:
won't that resolve to `cast/clients/clients`? Feels redundant.

Reply:
No, this will properly save them in cast/clients, but the name of the file in `date` folder will be `clients`

Reply:
<img width="1165" alt="image" src="https://github.com/WalletConnect/cast-server/assets/38994251/1b365aa6-4ec3-44c9-ab5b-1e4ed37b0c1c">

Reply:
We looked at it yesterday with @dnul and confirmed this is how we want it to look

Reply:
ok then :)